File: 1189-maximum-number-of-balloons/1189-maximum-number-of-balloons.cpp
class Solution {
public:
    int maxNumberOfBalloons(string text) {
        int arr[26];
        memset(arr,0,sizeof(arr));
        for(auto x:text)
        {
            arr[x-'a']++;
        }
        return min({arr[0],arr[1],arr[13],arr[11]/2,arr[14]/2});
    }
};

File: 1189-maximum-number-of-balloons/1189-maximum-number-of-balloons.java
class Solution {
    public int maxNumberOfBalloons(String text) {
        ArrayList<Integer> arr = new ArrayList<>(Collections.nCopies(26, 0));
        for(int i=0;i<text.length();i++)
        {
            int indx = text.charAt(i)-'a';
            arr.set(indx, arr.get(indx)+1);
        }
        return Math.min(arr.get(0),Math.min(arr.get(1),Math.min(arr.get(13),Math.min(arr.get(11)/2,arr.get(14)/2))));
    }
}